buckbuild.bzl

```diff
@@ -409,20 +409,22 @@ def get_aten_generated_files(enabled_backends):
 
 def get_aten_derived_type_src_rules(aten_rule_name, enabled_backends):
     return [
-        ":{}[{}]".format(aten_rule_name, "Register" + backend + ".cpp")
-        for backend in enabled_backends
-    ]
+        ":{}[{}]".format(aten_rule_name, "Register" + backend + "_0.cpp")
```

Comment:
While adjusting this, f-string might be a bit cleaner

Reply:
Starlark doesn't have f-strings -- https://github.com/bazelbuild/starlark/issues/91